docs/examples/tutorial/cython_tutorial/primes.pyx

```diff
@@ -1,9 +1,13 @@
 def primes(int nb_primes):
     cdef int n, i, len_p
     cdef int p[1000]
+
     if nb_primes > 1000:
         nb_primes = 1000
 
+
+
+
```

Comment:
If you do go the route of `p: cython.int[1000] = [0] * 1000` don't forget to remove this.

Reply:
Obviously, since I check the visuals before pushing, things like this will pop up immediately.